dune-project

```diff
@@ -1,2 +1,2 @@
-(lang dune 1.2)
+(lang dune 2.0)
 (name dns)
```

Comment:
```suggestion
(name dns)
(formatting disabled)
```

Dune 2.0 imposes some ocamlformat stuff, which we're not using here (yet?).

Reply:
Ah, I wasn't aware. Maybe we should?